### ISSUE TITLE
how do you spawn a working drone?

### ISSUE THREAD

**Issue by Kordeyne** [Aronnaxx/DroneCompanionsV2#1]
hi,
thank you so much for picking up this mod and trying to rework it.
i saw that you said crafting does not spawn the drones, but you alluded to manual spawning from the console working but iffy.
what command should i use to try to manually spawn a working drone?
i tried this one from the old mod page, but I just get a drone that does not do anything, just hovers in place.
is there a command currently that can spawn a working companion drone?

heading = Game.GetPlayer():GetWorldForward() 
offsetDir = ToVector3{heading.x, heading.y, heading.z} 
offsetDir.x = heading.x 
offsetDir.y = heading.y 
offsetDir.z = heading.z  
Game.GetCompanionSystem():SpawnSubcharacter(TweakDBID.new("DCO.Tier1Griffin1"),  2, offsetDir)

**Comment by Aronnaxx** [comment]
Sadly, the patch that occurred between upload and now busted it. I haven't gone back in to try any more fixes since the initial upload

**Comment by DigitalGameMachine** [comment]
> Sadly, the patch that occurred between upload and now busted it. I haven't gone back in to try any more fixes since the initial upload

Would be amazing if you could get this to work in 2025, many folks really like the concept of this mod.

**Comment by Superbug112** [comment]
> > Sadly, the patch that occurred between upload and now busted it. I haven't gone back in to try any more fixes since the initial upload
> 
> Would be amazing if you could get this to work in 2025, many folks really like the concept of this mod.

i agree. been scouring the net for a working drone companions mod. the idea of being able to craft them, for the system to be in the game almost like it was there when the game came out was a great experience for me and one that i used to use alot. will be keeping an eye on this in hopes that it gets worked on more.
